### ISSUE TITLE
Visually distinguish University Holidays from other dates

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/library_hours_rails#604]
Not too long ago, we visually distinguished dates designated as "Holidays" from other dates by adding a red background color to the day column. I'm guessing that treatment either failed accessibility checks for contrast, or we lost it in a Bootstrap update. 

It would be good to add it back in, or include some other way to visually distinguish these dates from "regular hours". Displaying a column header isn't quite enough.  

Here's how it looks today:
<img width="356" alt="image" src="https://github.com/user-attachments/assets/682baa3c-294f-4344-bd68-c0c0d226ba0d" />

**Comment by dbranchini** [comment]
https://www.figma.com/design/E6rFFnHIs7ch0oK5j9C1Wc/Tabular-view-and-Filtering?node-id=2040-6954&t=rZ0e9QJ1f5OgeizC-4

Design above, summarized as follows:
- Fog Light (#F4F4F4) background to holidays.
- Change font to Semi-bold (600) for better contrast/accessibility.

**Comment by saseestone** [comment]
Noting that we've had several requests from Access Services to use this display treatment for non-University Holidays like the ["End-Quarter Period"](https://studentservices.stanford.edu/calendar/academic-dates/stanford-academic-calendar-2024-2025#winter25) when libraries might be extended their hours for Finals.
